test/fixtures_redis_test.go

```
@@ -54,12 +54,10 @@ func TestFixturesRedisSuite(t *testing.T) {
 
 func (s FixturesRedisSuite) TestRedis() {
 	config := cfg.New()
-	config.Option(
+	_ = config.Option(
```

how dare you to ignore the error :P

---

In the `testing` universe, everything is allowed ;)

---

i guess it's going to fail anyway if something would have errored